apps/web/pages/event-types/[type]/index.tsx

```diff
@@ -473,6 +473,19 @@ const EventTypePage = (props: EventTypeSetupProps) => {
       }
     }
 
+    // Prevent two payment apps to be enabled
+    let enabledPaymentApps = 0;
+    for (const appKey in metadata?.apps) {
+      const app = metadata?.apps[appKey];
+      if (app.price && app.enabled) {
+        enabledPaymentApps++;
+      }
+
+      if (enabledPaymentApps > 1) {
```

Comment:
we should extract this logic in the appStore I think, create properly typed utils to get enabledApps and enabledPaymentApps